report/index.html

```diff
@@ -0,0 +1,344 @@
+      <header>
+        <div class="banner">
+          <div class="content">
+            <a href="https://allenai.org">
+              <img
+                src="https://cdn.jsdelivr.net/npm/@allenai/varnish@3.0.5/shellac/ai2.svg"
+                alt="Allen Institute for AI"
+              />
+            </a>
+          </div>
+        </div>
+        <div class="content header-columns">
+          <!-- TODO: add a logo for tango -->
+          <h2 class="header-title">Tango</h2>
+        </div>
+      </header>
+      <div id="chart"></div>
+    </div>
+
+    <!-- styling is not great, here we do some global css, but keep in mind that the actual viz is
+      an svg, so not all css makes sense -->
+    <style>
+      #chart {
+        max-width: var(--breakpoints-lg);
+        margin: 0 auto;
+        background: var(--palette-background-info);
+      }
+      #chart svg {
+        height: 100%;
+        width: 100%;
+      }
+      image:hover {
+        height: 44px;
+        transform: translate(0, -12px);
+      }
+      .node {
+        filter: drop-shadow(4px 8px 3px rgba(0, 0, 0, 0.5));
+      }
+      .node:hover {
+        filter: drop-shadow(5px 9px 4px rgba(0, 0, 0, 0.6));
+      }
+    </style>
+
+    <!-- include any libs you need here -->
+    <script src="https://unpkg.com/dayjs@1.10.7/dayjs.min.js"></script>
+    <script src="https://unpkg.com/dayjs@1.10.7/plugin/duration.js"></script>
+    <script src="https://unpkg.com/dayjs@1.10.7/plugin/relativeTime.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/5.9.7/d3.min.js"></script>
+    <script
+      src="https://unpkg.com/viz.js@1.8.2/viz.js"
+      type="javascript/worker"
+    ></script>
+    <script src="https://unpkg.com/d3-graphviz@2.6.1/build/d3-graphviz.min.js"></script>
+
+    <script>
+      //extend dayjs https://day.js.org/docs/en/durations/humanize
+      dayjs.extend(window.dayjs_plugin_duration);
+      dayjs.extend(window.dayjs_plugin_relativeTime);
+
+      let data;
+      let graphviz;
+
+      const states = {
+        COMPLETED: "completed",
+        FAILED: "failed",
+        RUNNING: "running",
+        INCOMPLETE: "incomplete",
+      };
+
+      // colors
+      const colors = {
+        B2: "#D5EAFE",
+        B6: "#265ED4",
+        B10: "#223367",
+
+        G6: "#1EC28E",
+
+        N2: "#F8F9FA",
+        N9: "#47515C",
+
+        O8: "#FF9100",
+
+        R6: "#F7605F",
+      };
+
+      // return the status in correct color and the duration if we have space
+      const formatStatus = (data, showDuration) => {
+        if (!data.status) {
+          return "";
+        }
+        let color = colors.N9;
+        let sep = " - after ";
+        switch (data.status) {
+          case states.COMPLETED:
+            color = colors.G6;
+            break;
+          case states.FAILED:
+            color = colors.R6;
+            break;
+          case states.RUNNING:
+          case states.RUNNING:
```

Comment:
```suggestion
          case states.RUNNING:
```

Or did you mean another state?

Reply:
oops

Reply:
fixed